Libraries/TestReport.psm1

```diff
@@ -392,20 +392,20 @@ Class TestSummary
 		}
 	}
 
-	[void] UpdateTestSummaryForCase([string]$TestName, [int]$ExecutionCount, [string]$TestResult, [string]$Duration, [string]$TestSummary, [object]$AllVMData)
+	[void] UpdateTestSummaryForCase([object]$TestData, [int]$ExecutionCount, [string]$TestResult, [string]$Duration, [string]$TestSummary, [object]$AllVMData)
 	{
 		if ( $this.AddHeader ) {
-			$this.TextSummary += "{0,5} {1,-65} {2,20} {3,20} `r`n" -f "ID", "TestCaseName", "TestResult", "TestDuration(in minutes)"
+			$this.TextSummary += "{0,5} {1, -10} {2,-40} {3,20} {4,20} `r`n" -f "ID", "TestArea", "TestCaseName", "TestResult", "TestDuration(in minutes)"
```

Comment:
You should check out long-length value of Area like PROD_CHECKPOINT or DYNAMIC_MEMORY. You may need more space in format.

Reply:
> You should check out long-length value of Area like PROD_CHECKPOINT or DYNAMIC_MEMORY. You may need more space in format.

@LiliDeng  was this a requested feature?
I think the plan was to stop changing the main LISAv2 framework and focus on other priorities, this not being one of them.
We're making changes that might break or require testing or further patching.

Reply:
@chvalean I know, this is for display a clear report for distro vendor (SUSE), I tested it, not break anything; for the summary change, as far as I see, I didn't see any pipeline using this report except the distro vendor pipeline, if you knows, please let know! Thanks for your concern.